packages/react/src/form/Form.tsx

```diff
@@ -92,9 +92,22 @@ const Form = React.forwardRef(function Form(
     extraProps: otherProps,
   });
 
+  const clearErrors = useEventCallback((name: string | undefined) => {
+    if (name && {}.hasOwnProperty.call(errors, name)) {
+      const nextErrors = { ...errors };
+      delete nextErrors[name];
+      onClearErrors(nextErrors);
+    }
+  });
+
   const contextValue: FormContext = React.useMemo(
-    () => ({ formRef, errors: errors ?? {}, onClearErrors }),
-    [formRef, errors, onClearErrors],
+    () => ({
+      formRef,
+      errors: errors ?? {},
+      onClearErrors,
```

Comment:
Do we still need `onClearErrors` on the context?
I can't see any other usage, removing it would avoid some confusion between two similarly named methods.

Reply:
Yeah good point! Will remove it.